requirements.txt

```diff
@@ -63,6 +65,7 @@ six==1.16.0
 sniffio==1.3.0
 sortedcontainers==2.4.0
 soupsieve==2.4.1
+SQLAlchemy==1.4.48
```

Comment:
This now causes a merge conflict, just the SQLAlchemy library though. Main has SQLAlchemy==2.0.15